### PR TITLE
Add new fields to Location model

### PR DESCRIPTION
**Pull request by mforner13**
This PR is the first of two which will replace the fields in the Location and Masterclass tables due to the following design changes:
- A masterclass will take place either remotely or in person
- Physical locations will be fetched from the google places API, rather than input manually

The three remote fields have been added to the masterclass model, rather than location because these will be individual to each masterclass, whereas multiple masterclasses can have the same location.

I have included the boolean `is_remote` to the masterclass model because remote and in person masterclasses will be displayed differently in the app.

**TODO**
- Update templates to display Location data from new fields
- Create another migration to remove the old Location fields

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -45,6 +45,12 @@ class Location(db.Model):
     street_name = db.Column(db.String(100), index=True)
     town_or_city = db.Column(db.String(50), index=True)
     postcode = db.Column(db.String(8), index=True)  # A postcode in the UK can't have more than 8 characters inc. space
+    is_remote = db.Column(db.Boolean, index=True)
+    remote_url = db.Column(db.String, index=True)
+    remote_joining_instructions = db.Column(db.String, index=True)
```
Comment (jonodrew) [inline app/models.py:50]: Not sure this belongs here

Comment (jonodrew) [reply app/models.py:50]: If we want locations to be re-useable maybe move this to the actual Masterclass? As it'll be unique data associated with each Masterclass

**Review by jonodrew — CHANGES_REQUESTED** [review]

**Comment by mforner13** [comment]
@jonodrew I have updated the model changes as we discussed last week

**Review by jonodrew — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**